### ISSUE TITLE
Error parsing ceph.spec: Failed to append PatchXXXX: lines

### ISSUE THREAD

**Issue by ktdreyer** [redhat-openstack/rdopkg#15]
I saw this crash when trying `rdopkg patch` with ceph:

```
Traceback (most recent call last):
  File "/usr/bin/rdopkg", line 9, in <module>
    load_entry_point('rdopkg==0.28.1', 'console_scripts', 'rdopkg')()
  File "/usr/lib/python2.7/site-packages/rdopkg/shell.py", line 96, in main
    runner.engage()
  File "/usr/lib/python2.7/site-packages/rdopkg/core.py", line 160, in engage
    new_args = self.action_manager.run_action(step, self.args)
  File "/usr/lib/python2.7/site-packages/rdopkg/action.py", line 155, in run_action
    return action_fun(*fun_args)
  File "/usr/lib/python2.7/site-packages/rdopkg/actions.py", line 801, in update_patches
    spec.set_new_patches(patch_fns)
  File "/usr/lib/python2.7/site-packages/rdopkg/utils/specfile.py", line 270, in set_new_patches
    error="Failed to append PatchXXXX: lines")
rdopkg.utils.exception.SpecFileParseError: Error parsing .spec file 'ceph.spec': Failed to append PatchXXXX: lines
```

**Comment by ktdreyer** [comment]
Here's my spec file: https://gist.github.com/ktdreyer/459942b227128c040861

To cause `rdopkg patch` to crash, I deleted line 83 (`Patch52:	0052-mon-add-check-to-CrushTester-test_with_crushtool.patch`), then ran "`rdopkg patch`". My `-patches` branch does have that final patch on it.

In order to work around this crash, I deleted *all* the existing "Patch" entries in my .spec file, and then `rdopkg patch` was able to successfully write them all into the .spec.
